src/DynamoSandbox/WebApp/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+</head>
+
+<body>
+  <script>
+    document.querySelector('body').style.overflow='hidden'
+  </script>
+  <div id="root"></div>
+<script>mainJs</script></body>
+
+
+
+</html>
```

Comment:
We should be able to remove this file as well as the js files

Reply:
removed